### DISCUSSION TITLE
resolution of micros

### DISCUSSION THREAD

**Discussion by herbkonik** [dbuezas/lgt8fx#292] [Q&A] [answered]
Hi,
using micros at an Atmega328P board the resolution is 4 micros (if i understand the arduino reference right).
Dos the LGT boards have the same resolution or is it different ?

**Comment by dbuezas** [comment]
At 32Mhz (no prescaler), the resolution of micros is 2 us.

See here:
* https://github.com/dbuezas/lgt8fx/blob/df986b57d2424452255e5ec975149948ae1ae7de/lgt8f/cores/lgt8f/wiring.c#L103
* https://github.com/dbuezas/lgt8fx/blob/df986b57d2424452255e5ec975149948ae1ae7de/lgt8f/cores/lgt8f/Arduino.h#L110
* And the fact that F_CPU is 32,000,000 (when no prescaler is set)

**Comment by dbuezas** [comment, marked as answer]
At 32Mhz (no prescaler), the resolution of micros is 2 us.

See here:
* https://github.com/dbuezas/lgt8fx/blob/df986b57d2424452255e5ec975149948ae1ae7de/lgt8f/cores/lgt8f/wiring.c#L103
* https://github.com/dbuezas/lgt8fx/blob/df986b57d2424452255e5ec975149948ae1ae7de/lgt8f/cores/lgt8f/Arduino.h#L110
* And the fact that F_CPU is 32,000,000 (when no prescaler is set)

**Reply by herbkonik** [reply]
Ah, OK, thank you !
